lib/scientist/experiment.rb

```diff
@@ -269,4 +269,16 @@ def try(name = nil, &block)
   def use(&block)
     try "control", &block
   end
+
+  def raise_on_mismatches=(bool)
+    @raise_on_mismatches = bool
+  end
+
+  def raise_on_mismatches?
+    if @raise_on_mismatches.nil?
+      self.class.raise_on_mismatches?
+    else
+      @raise_on_mismatches
```

Comment:
If we went with `attr_accessor` then I would convert this to the accessor instead of the instance variable.  Could prepend with `!!`.